src/ziggurat/server/routes.clj

```diff
@@ -27,6 +27,7 @@
       (conj (vec (concat actor-routes (get-routes))))
       (bidi/make-handler)
       (m/wrap-hyphenate)
+      (m/enable-swagger)
```

Comment:
can we follow middleware naming standard here? i.e. `wrap-swagger` ?

Reply:
Done